e2e-tests/scheduled-backup/conf/some-name-rs0-3.yml

```diff
@@ -0,0 +1,99 @@
+apiVersion: psmdb.percona.com/v1
+kind: PerconaServerMongoDB
+metadata:
+  name: some-name
+spec:
+  #platform: openshift
+  image:
+  imagePullPolicy: Always
+  backup:
+    enabled: true
+    image: perconalab/percona-server-mongodb-operator:1.1.0-backup
+    resources:
+      requests:
+        cpu: "300m"
+        memory: "0.5G"
+    storages:
+      aws-s3:
+        type: s3
+        s3:
+          credentialsSecret: aws-s3-secret
+          region: us-east-1
+          bucket: operator-testing
+          prefix: psmdb-scheduled-backup
+          insecureSkipTLSVerify: false
+      minio:
+        type: s3
+        s3:
+          credentialsSecret: minio-secret
+          region: us-east-1
+          bucket: operator-testing
+          endpointUrl: http://minio-service:9000/
+          insecureSkipTLSVerify: false
+      gcp-cs:
+        type: s3
+        s3:
+          credentialsSecret: gcp-cs-secret
+          region: us-east-1
+          bucket: operator-testing
+          endpointUrl: https://storage.googleapis.com
+          prefix: psmdb-scheduled-backup
+          insecureSkipTLSVerify: false
+      azure-blob:
+        type: azure
+        azure:
+          container: operator-testing
+          prefix: psmdb-scheduled-backup
+          credentialsSecret: azure-secret
+    tasks:
+    - name: every-min-minio
+      enabled: true
+      backupType: physical
```

Comment:
please add the example of this option into cr.yaml

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1112/commits/e8f6ce8911e7c1deb5c4e53815b85a2f56da7c98